hathor/indexes/tx_group_index.py

```diff
@@ -49,7 +49,7 @@ def _get_from_key(self, key: KT) -> Iterable[bytes]:
         raise NotImplementedError
 
     @abstractmethod
-    def _get_sorted_from_key(self, key: KT) -> Iterable[bytes]:
+    def _get_sorted_from_key(self, key: KT, tx: Optional[BaseTransaction] = None) -> Iterable[bytes]:
```

Comment:
Maybe add info about what is this new argument in the docstring? Or maybe a more expressive name than `tx`

Reply:
Improved the argument name and docstring. What do you think? https://github.com/HathorNetwork/hathor-core/pull/978/commits/57558a502daa480f03a942e50f8c5861fc676ff1